spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-basic-sample/src/test/java/com/example/DatastoreBookshelfExampleTests.java

```diff
@@ -64,6 +77,17 @@ public static void checkToRun() {
 				System.getProperty("it.datastore"), is("true"));
 	}
 
+	@Test
+	public void testSerializedPage() {
+		Book book = new Book("Book1", "Author1", 2019);
+		book.id = 12345678L;
+		this.bookRepository.save(book);
+		Awaitility.await().atMost(15, TimeUnit.SECONDS)
+				.until(() -> this.bookRepository.count() == 1);
+		assertThat(sendRequest("/allbooksserialized", null, HttpMethod.GET))
+				.contains("content\":[{\"id\":12345678}],\"pageable\":");
```

Comment:
Can you also verify the cursor?

Reply:
Unfortunately the cursor isn't actually part of the response. 
While the `DatastorePageable` type is what gets serialized into JSON, what gets written is just the base `Pageable` and doesn't have the cursor.

Reply:
fixed :)